crdppf/static/js/Crdppf/main.js

```diff
@@ -34,23 +34,25 @@ Ext.onReady(function() {
             for (var i = 0; i < buttons.length; i++){
                  buttons[i].addClass('msgButtonStyle'); 
             }
-            Ext.Msg.show({
-               title: Crdppf.labels.disclaimerWindowTitle,
-               msg: Crdppf.labels.disclaimerMsg,
-               buttons: Ext.Msg.YESNO,
-               fn: redirectAfterDisclaimer,
-               animEl: 'elId',
-               icon: Ext.MessageBox.WARNING
-            });
+            Crdppf.init_main(lang);
+
+            // Ext.Msg.show({
```

Comment:
Is commenting the whole `Ext.Msg.show` done on purpose ?

If yes, then we could deleted the whole section, no ?